File: Retos/Reto #0 - EL FAMOSO FIZZ BUZZ [Fácil]/javascript/joshArfDev.js
//Reto #0 
for (let number = 1; number <= 100; number++) {
  if ((number % 3 == 0) & (number % 5 == 0)) {
    console.log("fizzbuzz");
  } else if (number % 3 == 0) {
    console.log("fizz");
  } else if (number % 5 == 0) {  
    console.log("buzz");
  } else {
    console.log(number);
  } 
}

File: Retos/Reto #0 - EL FAMOSO FIZZ BUZZ [Fácil]/java/joshArfDev.java
public class joshArfDev {
    public static void main(String[] args) {

        System.out.println("FizzBuzz");

        //Inicializando un arreglo de 100 de length
        int rango[] = new int[100];

        //Rellenamos el arreglo del 1 al 100
        for(int i = 0; i < rango.length; i++){
            rango[i] = i + 1;
        }

        // Iteramos sobre el arreglo para imprimir resultados
        for (int numero : rango) {
            if (numero % 3 == 0 && numero % 5 == 0) {
                System.out.println("fizzbuzz");
            }
            else if (numero % 3 == 0) {
                System.out.println("fizz");
            }
            else if (numero % 5 == 0) {
                System.out.println("buzz");
            }
            else {
                System.out.println(numero);
            }
        }

    }
}